docs/src/webknossos-py/development.md

```diff
@@ -0,0 +1,89 @@
+# Contributions & Development
+
+## How to contribute
+
+We welcome community feedback and contributions! We are happy to have
+
+* general feedback and questions on the [image.sc forum](https://forum.image.sc/tag/webknossos),
+* feature requests and bug reports as [issues on GitHub](https://github.com/scalableminds/webknossos-libs/issues/new),
+* documentation, examples and code contributions as [pull requests on GitHub](https://github.com/scalableminds/webknossos-libs/compare).
+
+
+## Development
+
+The [webknossos-libs repository](https://github.com/scalableminds/webknossos-libs) is structured as a mono-repo, containing multiple packages:
+
+* `cluster_tools`
+* `webknossos`
+* `wkcuber`
+* (`docs`, see below for **Documentation**)
+
+See below for specifics of the different packages. Let's have a look at the common tooling first:
+
+* [**poetry**](https://python-poetry.org) is used for dependency management and publishing.
+  By default, it creates a [virtual environment](https://docs.python.org/3/tutorial/venv.html) for each package.
+  To run commands inside this package, prefix them with `poetry run`, e.g. `poetry run python myscript.py`,
+  or enter the virtual environment with `poetry shell`.
+  The creation of a separate environment can be disabled (e.g. if you want to manage this manually),
+  [see here for details](https://python-poetry.org/docs/configuration/#virtualenvscreate).
+  To install the preferred version for this repository, run
+  [`pip install -f requirements.txt`](https://github.com/scalableminds/webknossos-libs/blob/master/requirements.txt)
+* **CI**: We use continuous integration with [github actions](https://github.com/scalableminds/webknossos-libs/actions),
+  please see the `CI` workflow for details.
+* **Tooling** we try to use across projects (still WIP, see [#581](https://github.com/scalableminds/webknossos-libs/issues/581)):
+    * `format.sh`: black and isort
+    * `lint.sh`: pylint
+    * `typecheck.sh`: mypy
+    * `test.sh`: pytest and custom scripts
+* **Releases** are triggered via the
+  [`Automatic release` github action](https://github.com/scalableminds/webknossos-libs/actions/workflows/release.yml),
+  using the `Run workflow` button in the topright corner.
+  This updates the changelog and pushes a new tag, which triggers another CI run building and publishing the package.
+
+
+### webknossos package
```

Comment:
```suggestion
### webKnossos package
```

Reply:
Alternatively, maybe use lowercase and backticks when referring to the package name itself